### PR TITLE
Linking guide on how to create an Azure AD application in the Readme

### PR DESCRIPTION
**Pull request by JonathanCrd**
Addressing issue https://github.com/Azure/autorest.typescript/issues/1138.

### CODE REVIEW

**Comment by JonathanCrd** [comment]
@qiaozha I added a line in the readme referring to the guide, please let me know if you want me to add something else.

Also, I was thinking about putting the following sentence in bold like this: 

> You will also need to **register a new AAD application and grant access to {{ serviceName}}** by assigning the suitable role to your service principal (note: roles such as `"Owner"` will not grant the necessary permissions) ...

What do you think?

**Comment by qiaozha** [comment]
@JonathanCrd Thanks. I think the link part looks good and I like the idea of putting that sentence in bold.

**Comment by qiaozha** [comment]
@JonathanCrd You probably need to update the generated folder in the integration test and smoke test ?

**Review by sarangan12 — APPROVED** [review]
Looks Good. Please close the original issue with the comment as Issue Fixed

File: src/generators/static/README.md.hbs [hunk 1]
```diff
@@ -55,9 +55,11 @@ You can authenticate with Azure Active Directory using the [Azure Identity libra
 npm install @azure/identity
 ```
 
-You will also need to register a new AAD application and grant access to {{ serviceName}} by assigning the suitable role to your service principal (note: roles such as `"Owner"` will not grant the necessary permissions).
+You will also need to **register a new AAD application and grant access to {{ serviceName}}** by assigning the suitable role to your service principal (note: roles such as `"Owner"` will not grant the necessary permissions).
 Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables: `AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, `AZURE_CLIENT_SECRET`.
 
+For more information about how to create an Azure AD Application check out [this guide](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal).
```
Comment (ramya-rao-a) [inline src/generators/static/README.md.hbs:61]: Remove the locale i.e. `en-us` from the url here to allow people at different locations to fall back to their own locale

Comment (JonathanCrd) [reply src/generators/static/README.md.hbs:61]: Good catch, I opened PR https://github.com/Azure/autorest.typescript/pull/1155 for this.

**Pull request merged**